From 56a54aca1dae9f01a6aeec33c6d6dcd418157a39 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Thu, 23 Aug 2018 06:29:19 +0300
Subject: [PATCH] Makefile: remove ASIC_BIN_DIR support

No need for goal hl-asic-install since we release
through Jenkins now
---

diff --git a/Makefile b/Makefile
index 7bf0848..019ba03 100644
--- a/Makefile
+++ b/Makefile
@@ -17,7 +17,6 @@
 	@echo 'Install:'
 	@echo '  optional environment variables:'
 	@echo '    PLDM_HEX_DIR       - Palladium directory to install flash image'
-	@echo '    ASIC_BIN_DIR       - ASIC directory to install binaries'
 	@echo '  dali-<board>-install - Install to <board> e.g. onix, pldm, asic'
 	@echo '                           set DALI_INSTALL_LAB_IP=<lab IP> to'
 	@echo '                             override <board> default lab IP'
@@ -58,7 +57,7 @@
 .PHONY: hl-check-release
 hl-check-release:
 ifndef HL_RELEASE_VER
-	$(error HL_RELEASE_VER is undefined)
+	$(error HL_RELEASE_VER is undefined, usually looks like x.y.z)
 endif
 
 #TODO: add btl once it is separated from br-ext
@@ -174,21 +173,8 @@
 	@mkdir -p $@
 dali-pldm-install: ${PLDM_HEX_DIR}
 	-cp $(IMAGES_DIR)/$(HL_DEV)-spi-flash.hex $</
-ASIC_BIN_DIR ?= /tools/hl_shared_dir/production/embedded/head
-${ASIC_BIN_DIR}:
-	@mkdir -p $@
-.PHONY: hl-asic-install
-hl-asic-install: $(ASIC_BIN_DIR)
-	-cp $(IMAGES_DIR)/$(HL_DEV)-fit.itb $</
-	-cp $(IMAGES_DIR)/$(HL_DEV)-spi-flash.bin $</
-	-cp $(IMAGES_DIR)/$(HL_DEV)-u-boot.bin $</
-	-cp $(IMAGES_DIR)/$(HL_DEV)-u-boot-spl.bin $</
-	-cp $(IMAGES_DIR)/hl-fw-loader $</
-ifneq ($(notdir $(realpath $(ASIC_BIN_DIR))), head)
-	ln -nfs $$(basename $(ASIC_BIN_DIR)) $$(dirname $(ASIC_BIN_DIR))/latest
-endif
 dali-asic-install:
-	@$(MAKE) hl-asic-install
+	@
 
 .PHONY: dali-distclean gaudi-distclean
 dali-distclean gaudi-distclean:
